Index.html

```diff
@@ -2,42 +2,82 @@
     <title>Home</title>
     <head>
         <link rel="stylesheet" href="css/styles.css">
-        <script type=text/javascript src="js/loginbuttonlogic.js"></script>
     </head>
     <body>
-        <div id="overlay" class="overlay">
-            <div id="logInModal" class="modal">
-                <div class="logIn-content">
-                  <span class="close">&times;</span>
-                  <p>Some text in the Modal..</p>
-                </div>
-              
-              </div>
+        <div class="overlay">
+            <div class="modal">
+                <span class="close-modal">&times;</span>
+                <p>
+                    It's in theaters now! Coming this summer...
+                </p>
+                <p>
+                    Two brothers. In a van. And then a meteor hit. And they ran as fast as they could. From giant cat-monsters. And then a giant tornado came. And that's when things got knocked into twelfth gear...
+                </p>
+                <p>
+                A Mexican...armada shows up. With weapons made from to- tomatoes. And you better betch'ur bottom dollar that these two brothers know how to handle business.
+                </p>
+            </div>
         </div>
         <!-- THIS IS THE NAVBAR, YOU CAN COPY PASTE THIS CODE ONTO ANY PAGE-->
         <div class="topnav">
-            <input type="button" id="logInButton" value="Log In" class="logInButton" onClick="openLogin();" >
+            <input type="button" id="logInButton" value="Log In" class="logInButton">
```

Comment:
there's nothing wrong with using the `onClick` attribute, but, in my ~Professional Opinion~, it's better to get in the habit of doing all your event binding (aka adding functions to handle clicking, hovering, etc) entirely in your JS files.